images/e2e-dind-nodejs/init.sh

```diff
@@ -40,4 +43,5 @@ if [[ "$K3D_ENABLED" == "true" ]]; then
   fi
   k3d cluster create k3d "${ARGS[@]}"
 fi
-bash -c "$@"
\ No newline at end of file
+bash -c "$@"
+}
```

Comment:
```suggestion
```
I can't see any open bracket for this one. I think it's not needed.

Reply:
Ye it was a typo. Ty